src/Transformer/AnnotationTransformer/AnnotationDefinitionInterface.php

```diff
@@ -0,0 +1,60 @@
+<?php declare(strict_types=1);
+
+namespace SilverStripe\Config\Transformer\AnnotationTransformer;
+
+use ReflectionClass;
+use ReflectionMethod;
+
+interface AnnotationDefinitionInterface
```

Comment:
Might be worth a doc block to explain the purpose of classes implementing this interface